src/components/Coding.jsx

```diff
@@ -0,0 +1,10 @@
+import Articles from "./Articles";
```

Comment:
Similar to the above, we should be able to reuse the same component that renders all of our articles to pass an optional query to axios or fetch all articles if no query is present (depending on what the url looks like) Take a look at how we can clean this up and make the code more DRY and future proof 👍